eventdriven/best-practices.md

```diff
@@ -84,82 +84,16 @@ Use a centralized schema management registry (e.g. Confluent Schema Registry or
 In addition to the major version given by topic/queue name, the full version of the API should be written into the message header. 
 Use the header field `x-api-version` and write the version in [SemVer](https://semver.org/) notation.
 
-### Reflect deprecation in documentation
```

Comment:
I would not delete these best practices. If someone is really interested in writing good APIs it can help. Everyone else won't read it anyway. :)